conversions/Backwards/src/main/java/com/agentdid127/resourcepack/backwards/impl/ModelConverter.java

```diff
@@ -54,233 +56,225 @@ protected void findFiles(Path path) throws IOException {
             if (file.isDirectory()) {
                 remapModelJson(Paths.get(file.getPath()));
                 findFiles(Paths.get(file.getPath()));
-
             }
         }
     }
 
     /**
      * Updates model Json to newer versions
+     * 
      * @param path
      * @throws IOException
      */
     protected void remapModelJson(Path path) throws IOException {
+        if (!path.toFile().exists())
+            return;
+        Files.list(path).filter(path1 -> path1.toString().endsWith(".json")).forEach(model -> {
+            try {
+                JsonObject jsonObject;
+                if (Util.readJson(packConverter.getGson(), model) != null && Util.readJson(packConverter.getGson(), model).isJsonObject())
+                    jsonObject = Util.readJson(packConverter.getGson(), model);
+                else {
+                    if (packConverter.DEBUG) {
```

Comment:
This is a logic error. Needs to convert from 1.20 backwards (or else features will not update)